docs/main/docs/collator/testnet/testnet-collator.md

```diff
@@ -0,0 +1,214 @@
+---
+sidebar_position: 1
+---
+
+# Run a t0rn Testnet Collator
+
```

Comment:
similar NOTE here that we will be updating our bootnode and release version once we have revamped the testnet